.gitignore

```diff
@@ -1 +1,6 @@
 node_modules
+package-lock.json
```

Comment:
It's ok to check in your package-lock. It's actually a good idea to do so so you can use `npm ci` in your build step for circle.

Reply:
Hey Toby!
Got it! I will not include package-lock.json in my git ignore.